packages/rest-client/src/services/store/store.ts

```diff
@@ -58,6 +58,25 @@ export function insert(schema: string, data: Resource[]): void {
   }
 }
 
+/**
```

Comment:
in the last PR I said this store stuff was going on pause (and it is!) so this isn't super necessary but I just threw it in because it was quick and sets up some of the code that will be used at a later date 😄